### PR TITLE
0.8: Remove experimental bits for 0.8 release

### PR DESCRIPTION
**Pull request by DrusTheAxe**

### CODE REVIEW

File: build/NuSpecs/AppxManifest.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <Identity Name="Microsoft.ProjectReunion.0.5" ProcessorArchitecture="x64" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.52102.3000.0" />
```
Comment (huichen123) [inline build/NuSpecs/AppxManifest.xml:3]: >5 [](http://example.com/codeflow?start=45&length=1)

if you change below, it's probably better to keep this consistent.

**Comment by huichen123** [comment]
>   </PropertyGroup>

not needed

---
Refers to: build/NuSpecs/ProjectReunion-Nuget-Native.C.props:7 in 809b65b. [](commit_id = 809b65b982132456421f924ae7248b17a915d2df, deletion_comment = False)

File: build/NuSpecs/ProjectReunion-Nuget-Native.targets [hunk 2]
```diff
@@ -9,18 +9,4 @@
     <Native-Platform Condition="'$(Platform)' != 'Win32'">$(Platform)</Native-Platform>
   </PropertyGroup>
```
Comment (huichen123) [inline build/NuSpecs/ProjectReunion-Nuget-Native.targets:10]: are these needed?

File: build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props [hunk 3]
```diff
@@ -6,20 +6,4 @@
     <_ProjectReunionFoundationPlatform Condition="'$(Platform)' != 'Win32'">$(Platform)</_ProjectReunionFoundationPlatform>
   </PropertyGroup>
```
Comment (huichen123) [inline build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props:7]: not needed

File: build/CopyFilesToStagingDir.ps1 [hunk 4]
```diff
@@ -79,27 +69,11 @@ if($PublishAppxFiles)
...
-# Libraries (*.lib)
-PublishFile $FullBuildOutput\ProjectReunion_BootstrapDLL\Microsoft.ProjectReunion.Bootstrap.lib $NugetDir\lib\win10-$Platform
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.lib $NugetDir\lib\win10-$Platform
-#
-# MSIX Framework package - DLLs
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.dll $NugetDir\runtimes\win10-$Platform\native
-PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.pdb $NugetDir\runtimes\win10-$Platform\native
-#
 # MSIX Main package
 PublishFile $FullBuildOutput\DynamicDependency.DataStore\DynamicDependency.DataStore.exe $NugetDir\runtimes\win10-$Platform\native
```
Comment (huichen123) [inline build/CopyFilesToStagingDir.ps1:78]: >DynamicDependency [](http://example.com/codeflow?start=57&length=17)

so bootstrap is experimental, but dd is not?

**Comment by huichen123** [comment]
if this is for 0.8, should we do it in 0.8 branch instead?

**Comment by axelandrejs** [comment]
> if this is for 0.8, should we do it in 0.8 branch instead?

Normally we go main->release, but here I agree that this makes sense to do directly in the release branch. We want to keep the experimental APIs in main.

File: build/CopyFilesToStagingDir.ps1 (hunk 4 above)
Comment (DrusTheAxe) [reply build/CopyFilesToStagingDir.ps1:78]: DynDep (which includes bootstrap) is experimental, but for the purpose of expediting the release the DataStore (PRmain) and DDLM (PRddlm) are untouched. The build will create but then we'll ignore those artifacts

Unless @BenJKuhn tells me it's easier to remove them here AND update the aggregator pipeline to not fail on the absence than to manually remove them from the final result

File: build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props (hunk 3 above)
Comment (DrusTheAxe) [reply build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props:7]: But not harmful either. I was focused on deleting what's experimental but leaving anything inert as-is

@BenJKuhn which way do you want to go?

**Comment by DrusTheAxe** [comment]
> > if this is for 0.8, should we do it in 0.8 branch instead?
> 
> Normally we go main->release, but here I agree that this makes sense to do directly in the release branch. We want to keep the experimental APIs in main.

Doh! That's the intent. Thanks for spotting that. Let me see where to alter the PR...

File: build/NuSpecs/ProjectReunion-Nuget-Native.targets (hunk 2 above)
Comment (DrusTheAxe) [reply build/NuSpecs/ProjectReunion-Nuget-Native.targets:10]: See next comment?

**Comment by DrusTheAxe** [comment]
> Doh! That's the intent. Thanks for spotting that. Let me see where to alter the PR...

I don't see how I can retarget the PR to the release branch instead of main. @BenJKuhn  @axelandrejs @huichen123 @kythant any idea short of abandoning this and making a new PR?

File: Directory.Build.props [hunk 5]
```diff
@@ -6,11 +6,8 @@
 <Project InitialTargets="DirectoryBuildPropsInfo" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Condition="$(BuildingWithBuildExe) != 'true'">
     <IsTDPConfiguration>false</IsTDPConfiguration>
-    <IsTDPConfiguration Condition="'$(Configuration)' == 'Debug_test'">true</IsTDPConfiguration>
```
Comment (huichen123) [inline Directory.Build.props:9]: ><IsTDPConfiguration Condition="'$(Configuration)' == 'Debug_test'">true</IsTDPConfiguration> [](http://example.com/codeflow?start=4&length=92)

now we have unrelated changes

Comment (DrusTheAxe) [reply Directory.Build.props:9]: rebased to release/reunion-0.8

Comment (huichen123) [reply Directory.Build.props:9]: maybe better just start fresh

Comment (DrusTheAxe) [reply Directory.Build.props:9]: I'd rather not. It takes ~40 minutes to turn a build, plus however long we're queued. This change has no impact on the resulting Transport Package, nor even the build pipeline or inner-loop as we don't use Debug_test configs (hence the reason they've been stripped from main)

Comment (huichen123) [reply Directory.Build.props:9]: i doubt we want to include these changes to release branch.

**Comment by DrusTheAxe** [comment]
Bad rebase! Bad! No soup for you!

Replacing with a new PR to avoid pulling in unwanted changes from main
